src/ol_dbt/models/intermediate/micromasters/_int_micromasters__models.yml

```
@@ -2,6 +2,56 @@
 version: 2
 
 models:
+- name: int__micromasters__exam_grades_old_data
```

```suggestion
- name: int__micromasters__dedp_proctored_exam_grades
```

I agree with Rachel that this table name could be clearer. If this is too long, we could remove the word `proctored`